CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Twilio Voice React Native SDK has now reached milestone `beta.4`. Included in th
 - Fixed inconsistency with `AudioDevice` typings, preferring `undefined` over `null` for optional values.
 - Fixed an issue with `call.isMuted()` and `call.isOnHold()` APIs. They should now always return `boolean | undefined` instead of potentially returning `null`.
 - Fixed an issue with `call.getFrom()`, `call.getTo()`, and `call.getSid()` APIs. They should now always return `string | undefined` instead of potentially returning `null`.
+- Call Message feature for Android
```

Comment:
This is one of the most anticipated features in RN SDK. Let's put it under "Features" section, above "Fixes".

## Features

### Call Message Events (Beta)

// Add new information about the new API and some related links. Also add example.
// You can use this as a reference.